src/VisualizationTests/HelixWatch3DViewModelTests.cs

```diff
@@ -149,14 +149,14 @@ public void Node_RenderingUpToDate()
             p1.UpdateValue(new UpdateValueParams("IsVisible", "false"));
 
             Assert.True(BackgroundPreviewGeometry.HasNumberOfPointsCurvesAndMeshes(7, 6, 0));
-            Assert.AreEqual(6, BackgroundPreviewGeometry.NumberOfInvisiblePoints());
+            Assert.AreEqual(1, BackgroundPreviewGeometry.NumberOfInvisiblePoints());
```

Comment:
@marimano Are we just fixing the test to match the behavior here, or have the changes you made actually result in there being less objects in the scene?